### PR TITLE
RedisJson: when retrieving the hash the conversion from string to has was not correct and fix unit tests

### PR DESCRIPTION
**Pull request by aboimpinto**

### CODE REVIEW

File: src/NeoSharp.Core/Cryptography/ECPoint.cs [hunk 1]
```diff
@@ -51,10 +51,22 @@ private ECPoint()
         /// <param name="point">Point</param>
         public ECPoint(byte[] point)
         {
-            IsInfinity = false;
+            if (point == null)
```
Comment (shargon) [inline src/NeoSharp.Core/Cryptography/ECPoint.cs]: this is on other PR.

File: src/NeoSharp.Core/Messaging/MessageHandlerProxy.cs [hunk 2]
```diff
@@ -48,10 +48,10 @@ public async Task Handle(Message message, IPeer sender)
             await specificMessageHandler.Handle(specificMessage, sender);
             var completedAt = DateTime.UtcNow;
 
-            var handledWithin = (completedAt - startedAt).TotalMilliseconds;
+            var handledWithin = (completedAt - startedAt).TotalSeconds;
```
Comment (shargon) [inline src/NeoSharp.Core/Messaging/MessageHandlerProxy.cs]: why this change?

File: src/NeoSharp.Core/Cryptography/ECPoint.cs (hunk 1 above)
Comment (aboimpinto) [reply src/NeoSharp.Core/Cryptography/ECPoint.cs]: damm .... looks like I mistake the PR ... 
after merge the other I will solve this conflict.

File: src/NeoSharp.Core/Messaging/MessageHandlerProxy.cs (hunk 2 above)
Comment (aboimpinto) [reply src/NeoSharp.Core/Messaging/MessageHandlerProxy.cs]: Because is readable in seconds for humans
with second we will get the 99.9999 and it's easier to read in the logs.

Comment (shargon) [reply src/NeoSharp.Core/Messaging/MessageHandlerProxy.cs]: Usually this should be very fast so we need the time in ms ... if we have some message who spend more than 1 second, we have something very wrong

Comment (aboimpinto) [reply src/NeoSharp.Core/Messaging/MessageHandlerProxy.cs]: well .. that is the reason to change .... to popup the problem. It's easy to see by human eye

**Comment by shargon** [comment]
I removed the log line because this should be in other PR, we need to fix the unit tests first.
